### ISSUE TITLE
[Bug]:Using a bee jar on a leashed bee crashes the client upon its release (And deletes parts of the inventory) 

### ISSUE THREAD

**Issue by Porcy** [Resourceful-Bees/ResourcefulBees#357]
### Bug Description

I used to capture my bees while they're attached to a fence post after breeding them, for easier capture. When releasing those bees later, the client crashes and the last few things added in my inventory just disappear upon login in back.  

### How to Reproduce?

1. Attach a bee to a fence post using a lead.
2. Capture it in a bee jar
3. Release the bee far away from the original fence post (I presume the distance is involved)
4. Enjoy your client crash and loss of items

### Expected Behavior

Bee released with maybe the lead falling off, or not being able to capture a leashed bee.

### Version

0.10.7

### Mod Loader Version

1.16.5 - 36.2.34

### Acknowledgement

- [X] I agree that I read the terms at the top of the page.

**Comment by ThatGravyBoat** [comment]
crash report please

**Comment by Porcy** [comment]
Yeah, that seems a bit more complex to reproduce than what I tought. I'll investigate further to find the exact steps needed. Happened to me 4 times on Enigmatica 6 Expert so it's not a fluke.

**Comment by Epic428** [comment]
Well if you crashed you should have a crash report along with a log file in your logs folder. You could just send us one of those files so we can get a stack trace to have a starting point

**Comment by Porcy** [comment]
I don't think it generated a crash report. The client terminates the connexion but I can still login right after, with a rollback of sort on the content of the inventory reverted to before the bug. This happened yesterday and no new crash report have been created in minecraft/crash-reports folder. It looks to me like a way to prevent a server crash. I suspect nbt corruption because as far as I can tell, the bee jars have been stored for a time into backpacks from Sophisticated Backpacks mod, stored themselves into chests. I'll investigate further when I have time.

**Comment by Epic428** [comment]
The client may have something in the logs but yeah if you can find a reliable way to reproduce the issue that’ll be helpful too

**Comment by Porcy** [comment]
Went through my logs and tried reproducing for a solid hour. Nothing. I'll update if I get the bug again and I'll check the logs then.
